config/environments/test.rb

```diff
@@ -15,11 +15,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  if ENV['CODECOV_TOKEN']
-    config.eager_load = true
-  else
-    config.eager_load = false
-  end
+  config.eager_load = true
```

Comment:
Yeah, that won't work, we only want eager loading when the CI environment variable is set, i.e. not running tests in your local env (check the comment above).